mne/viz/_3d.py

```diff
@@ -304,7 +304,8 @@ def _set_aspect_equal(ax):
 
 @verbose
 def plot_evoked_field(evoked, surf_maps, time=None, time_label='t = %0.0f ms',
-                      n_jobs=1, fig=None, verbose=None):
+                      n_jobs=1, fig=None, vmax=None, n_contours=21,
```

Comment:
21 was the mayavi default?

Reply:
yes, IIRC

Reply:
My bad, [it's actually 10](https://github.com/enthought/mayavi/blob/9c128635943e28513776dec68492ffd0268c28d1/tvtk/tools/mlab.py#L771), 21 was the number of contours used in the example. I will change that.